src/main/java/duke/command/DoneCommand.java

```diff
@@ -0,0 +1,78 @@
+package duke.command;
+
+import duke.SaveManager;
+import duke.TaskManager;
+import duke.Ui;
+import duke.exception.DukeInputException;
+import duke.task.Task;
+
+/**
+ * Represents user command to mark a <code>Task</code> as done.
+ */
+public class DoneCommand extends ComplexCommand {
```

Comment:
Should the class name be a verb? Pardon my naming sense, but do consider something like DoneMarker